### PR TITLE
Feature/styles

### PR DESCRIPTION
**Pull request by Dandush03**
# Milestone 3

Finished styling the element and adedded props styling future

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "import/no-unresolved": 0
```
Comment (bellom) [inline .eslintrc.json:26]: Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)
You need to fix linter's error.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
-# React-Calculator
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (bellom) [inline README.md:1]: **Kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

File: src/component/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      this.width = '50%';
+    }
+
+    if (props.color) {
+      this.color = props.color;
+    }
+
+    this.style = {
+      width: this.width,
+      'background-color': this.color,
```
Comment (bellom) [inline src/component/Button.jsx:22]: I'm curious, were you trying to write `'background-color'` or `background-color`?
I think it should be without quote right

**Comment by Dandush03** [comment]
- [x] Deleted commented line
- [x] Fix readme
- [x] Fix EsLint file
- [ ] Modified BG Color Attribute: The issue with this one is that since this attribute has a '-' JS detect it as it's been substracted so I would have to keep the quotes, now if I use the property just 'background' without the '-color' then it would set all other properties of background to none and it would look ugly

**Pull request merged**